### PR TITLE
Made everything better

### PR DESCRIPTION
**Pull request by PerfectSeiha**

### CODE REVIEW

**Comment by WaximeA** [comment]
idk if it's usefull ;)

File: foo.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Hello the world, i'm Waxime !  #
-We’re trying git at _School_! 
-This is a practical work created by the fabulous *@Nek-*  <3.
\ No newline at end of file
+#### We’re trying git at _School_! 
+#### This is a practical work created by the fabulous *@Nek-*  <3333333333333.
+#### XOXOXOXOXOXOXO :33333333333333
```
Comment (WaximeA) [inline foo.md]: change xoxoxo

Comment (PerfectSeiha) [reply foo.md]: XOs changed

Comment (WaximeA) [reply foo.md]: tks

**Pull request merged**